content/blog/2020/08/camel-quarkus-release-1.0.0/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Camel Quarkus 1.0.0 Released"
+date: 2020-08-10
+authors: ["ppalaga"]
+categories: ["Releases", "Camel Quarkus"]
+preview: What's new in Camel Quarkus 1.0.0
+---
+
+<sub><sup>Original image by <a href="https://commons.wikimedia.org/wiki/User:99of9">Toby Hudson</a> <a href="https://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a> via <a href="https://en.wikipedia.org/wiki/Camel_racing#/media/File:CamelRacingCamelCup2009Heat.JPG">Wikipedia</a></sup></sub>
```

Comment:
Might make sense to add [manual summary](https://gohugo.io/content-management/summaries/#front-matter-summary), this bit ends up in the [blog post list](https://deploy-preview-454--camel.netlify.app/blog/).

Reply:
Let me add a summary